packages/ui/styles/index.css

```diff
@@ -119,7 +119,7 @@ input[type='number']::-webkit-outer-spin-button {
     --text-secondary: rgba(var(--v2-dark), 0.7);
     --text-tertiary: rgba(var(--v2-dark), 0.5);
     --text-brand: rgba(var(--v2-brand), 0.95);
-    --text-interactive-disabled: rgba(var(--v2-light), 0.2);
+    --text-interactive-disabled: rgba(var(--v2-dark), 0.2);
```

Comment:
@NoahSaso feel free to double check this

Reply:
Hmm, I think both themes use `--v2-light` (looking at the css on the bottom right).

<img width="562" alt="Screen Shot 2022-08-19 at 11 12 15 PM" src="https://user-images.githubusercontent.com/6721426/185731681-e7f0e0bc-3c83-4ff9-b01f-ec3c0c42ed62.png">
<img width="563" alt="Screen Shot 2022-08-19 at 11 12 43 PM" src="https://user-images.githubusercontent.com/6721426/185731701-fdf414c6-969b-487b-bb7f-3cbff782291f.png">

Reply:
Never mind, I think you're right :)